<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowCamera</class>
 <widget class="QMainWindow" name="MainWindowCamera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simox - VirtualRobot - Show Robot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frameViewer">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButtonReset">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonLoad">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>20</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Select Robot File</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>391</width>
         <height>131</height>
        </rect>
       </property>
       <property name="title">
        <string>Joints</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Joint Value</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcdNumberJointValue">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>70</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="smallDecimalPoint">
         <bool>true</bool>
        </property>
        <property name="digitCount">
         <number>5</number>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderPos">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>80</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="labelMinPos">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>100</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>-1.57</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelMaxPos">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>100</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>1.57</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxJoint">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>40</y>
          <width>181</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxRobotNodeSet">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>181</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Select RobotNodeSet</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>20</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Select RobotNode</string>
        </property>
       </widget>
      </widget>
      <widget class="QGraphicsView" name="cam2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>520</y>
         <width>320</width>
         <height>240</height>
        </rect>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
      <widget class="QGraphicsView" name="cam1">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>260</y>
         <width>320</width>
         <height>240</height>
        </rect>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1114</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
